_locales/en/messages.json

```diff
@@ -233,14 +233,14 @@
   "extensionHasUpdated": {
     "message": "Scratch Addons has updated to v$1"
   },
-  "extensionUpdateInfo1_v1_17": {
-    "message": "New addon: \"old studio layout\", which reverts the layout of the studio page back to the one used before the studio update. Go to $1 to enable it."
+  "extensionUpdateInfo1_v1_18": {
+    "message": "New addon: \"turbowarp button\", which adds a button on project pages to play the project in TurboWarp.org. Go to $1 to enable it."
```

Comment:
TurboWarp should be capitalized in the addon name and lowercase in the website.

Reply:
Nice catch. I'm keeping uppercase in domain to be consistent with the addon's description.